conf/gravitino.conf.template

```diff
@@ -31,8 +31,8 @@ gravitino.server.webserver.responseHeaderSize = 131072
 gravitino.entity.store = kv
 # The RocksDB entity store
 gravitino.entity.store.kv = RocksDBKvBackend
-# The RocksDB backend path for entity store
-# gravitino.entity.store.kv.rocksdbPath = /tmp/gravitino
+# The RocksDB backend path for entity store, default value is as below, please uncomment and change it in your production environment.
+# gravitino.entity.store.kv.rocksdbPath = ${GRAVITINO_HOME}/data/rocksdb
```

Comment:
I think it will make people misunderstand that Gravitino can support `${xxx}`. You'd better avoid using this, maybe we can use relative path like `data/rocksdb`.

Reply:
Currently, I'm afraid we do not support the relative path here. Perhaps I need to support it here...

Reply:
@jerryshao 
Should we need to support it in release 0.4.0?

Reply:
you'd better support relative path.